python/lsst/ts/observatory/control/mock/mtcs_mock.py

```diff
@@ -0,0 +1,326 @@
+# This file is part of ts_observatory_control.
+#
+# Developed for the LSST Telescope and Site Systems.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+
+__all__ = ["MTCSMock"]
+
+import asyncio
+
+from .base_group_mock import BaseGroupMock
+
+from lsst.ts import salobj
+
+import astropy.units as u
+import numpy as np
+
+from astropy.time import Time
+from astropy.coordinates import AltAz, ICRS, EarthLocation, Angle
+
+from lsst.ts.idl.enums import MTPtg
+
+from ..utils import subtract_angles
+
+LONG_TIMEOUT = 30  # seconds
+HEARTBEAT_INTERVAL = 1  # seconds
+CLOSE_SLEEP = 5  # seconds
+
+
+class MTCSMock(BaseGroupMock):
```

Comment:
Would it make any sense to have a base class shared by ATCSMock and MTCSMock (that inherited fromBaseGroupMock)? I realize the components differ so there'd not be a huge overlap, but perhaps there is enough to be useful?